bytestring.cabal

```diff
@@ -208,4 +209,5 @@ benchmark bytestring-bench
                     bytestring,
                     deepseq,
                     tasty-bench,
-                    random
+                    random,
+                    ieee754
```

Comment:
We probably already vendor the few constants used from this package in the main library component. But I guess the plumbing to use them in the testsuite is not yet convenient. I might work on improving that soon.

Reply:
Is there an issue or pull request to reference this change in?